workflow/rules/calculate_run_background.smk

```diff
@@ -0,0 +1,34 @@
+# vim: syntax=python tabstop=4 expandtab
+# coding: utf-8
+
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule calculate_run_background:
+    input:
+        gvcfs=lambda wildcards: [
+            "snv_indels/mutect2_gvcf/%s_%s.merged.gvcf.gz" % sample_run for sample_run in get_units_per_run(units, wildcards)
+        ],
+    output:
+        background=temp("annotation/calculate_run_background/{run}_run_background.tsv"),
+    params:
+        type="run",
```

Comment:
should we maybe have something more descriptive then "run"? I started to wondering if this was the same as the wildcard (which it isn't)